src/ert/shared/ensemble_evaluator/narratives/proxy.py

```diff
@@ -39,6 +39,8 @@ async def handle_server(server, client, msg_q):
         async def handle_client(client, _path):
             msg_q = asyncio.Queue()
             if _path == "/client":
+                # pylint: disable=no-member
```

Comment:
Should we consider the following suggestion from the author of websockets: https://github.com/aaugustin/websockets/issues/940#issuecomment-874012438 ?

Reply:
Fixed, changing our code from `websockets.connect()` to `websockets.client.connect()`